rovercode/app.py

```diff
@@ -120,21 +120,24 @@ def on_open(ws):
     global rovercode_web_url
     global client_id
 
-    info = json.loads(session.get('{}/api/v1/rovers?client_id={}'.format(rovercode_web_url, client_id)).text)[0]
+    info = json.loads(session.get('{}/api/v1/rovers?client_id={}'.format(rovercode_web_url, client_id)).text)['results'][0]
```

Comment:
I know it wasn't doing it before, but it might be good to handle a possible exception here if the API doesn't respond. Something like an LED going red so that the user knows that it's not connected

Reply:
I definitely agree! I confirmed that this exception shows up well in the log, but we'll need something user-facing.

Do you mind if I do the red-LED in one of the upcoming PRs when I set up the LED?

Reply:
Sounds good to me